src/coreclr/jit/gentree.h

```diff
@@ -866,10 +866,11 @@ struct GenTree
                                                //             alignment of 1 byte)
 #define GTF_IND_INVARIANT           0x01000000 // GT_IND   -- the target is invariant (a prejit indirection)
 #define GTF_IND_ARR_INDEX           0x00800000 // GT_IND   -- the indirection represents an (SZ) array index
+#define GTF_IND_NONNULL             0x00200000 // GT_IND   -- the indirection never returns null (zero)
```

Comment:
Is there some reason you didn't use the "next available bit"?
```suggestion
#define GTF_IND_NONNULL             0x00400000 // GT_IND   -- the indirection never returns null (zero)
```

Reply:
yeah should work with 004, thanks